content/backmatter.tex

```diff
@@ -540,7 +549,8 @@ \subsection{\textit{C11} and \CorCpp: \FUNC{shmem\_fetch}, \FUNC{shmem\_set}, %%
 In addition, the abbreviated names ``cswap'', ``finc'', and ``fadd'' were
 expanded for clarity to ``compare\_swap'', ``fetch\_inc'', and ``fetch\_add''.
 
-\subsection{\textit{Fortran} API}\label{subsec:deprecate-fortran} %% WARNING: Issue #66.
+\subsection{\textit{Fortran} API} %% WARNING: Issue #66.
```

Comment:
Can we remove this comment? Looks like it may correspond to an issue you filed a few years ago - https://github.com/openshmem-org/specification/issues/66.

Reply:
I'd rather leave the comment. It's a visual reminder for why `\textit{Fortran}` is used instead of `\Fortran` (the PDF bookmark is broken when using commands that take optional arguments). This issue did come up once during the v1.5rc2 period.

Reply:
(To clarify, that issue was not resolved. We punted on it.)

Reply:
Could you add a little more context to clarify what the warning is about? Something like:
```
%% WARNING: Need to use \textit{Fortran} instead of \Fortran, see Issue #66 for details
```

Reply:
:+1: I clarified the problem at the bottom of the deprecated-rationale entries. It'll remind document editors when they add new entries to the list.

Reply:
Thanks! I was absolutely thinking of myself, that I would forget why we needed to handle the Fortran case in this way. :)